File: orchalanguage-spring-integration-implementation/src/main/java/orcha/lang/compiler/referenceimpl/springIntegration/SendInstructionForSpringIntegration.java
package orcha.lang.compiler.referenceimpl.springIntegration;

import orcha.lang.compiler.syntax.SendInstruction;

public class SendInstructionForSpringIntegration extends SendInstruction {

    public SendInstructionForSpringIntegration() {
        super();
    }

    public SendInstructionForSpringIntegration(String sendInstruction) {
        super(sendInstruction);
    }

    public SendInstructionForSpringIntegration(String sendInstruction, int lineNumber) {
        super(sendInstruction, lineNumber);
    }

    public String getVariables() {
        return "payload." + super.getVariables();
    }


}

File: orchalang-kafka-implementation/src/main/kotlin/orcha/lang/compiler/kafka/SendInstructionForSpringIntegration.kt
package orcha.lang.compiler.kafka

import orcha.lang.compiler.syntax.SendInstruction

class SendInstructionForSpringIntegration : SendInstruction {

    constructor() : super() {}

    constructor(sendInstruction: String) : super(sendInstruction) {}

    constructor(sendInstruction: String, lineNumber: Int) : super(sendInstruction, lineNumber) {}

    override fun getVariables(): String {
        return "payload." + super.getVariables()
    }


}
